docs/input4MIPs_datasets_CMIP6Plus.html

```diff
@@ -65,6 +65,44 @@
       <th>further_info_url</th>
     </tr>
   </tfoot>
+    <tr>
+      <td><a href='https://github.com/PCMDI/input4MIPs_CVs/blob/main/CVs/input4MIPs_source_id.json' target='_blank'>Registered in CVs</a></td>
+      <td>CMIP6Plus</td>
+      <td>CR-CMIP-0-2-0</td>
+      <td>0.2.0</td>
```

Comment:
@znichollscr did you want this to exist, even though this data was never made available to anyone? If we haven't published, or made this available through an alternate site/source, I would be inclined to remove it rather than listing it

Reply:
I just added the 'abandoned' publication status (https://github.com/PCMDI/input4MIPs_CVs/blob/126135a72d4a8a1a5709121233ffb4d3dcb6dfad/CVs/input4MIPs_publication_status.json#L14). That way, if anyone had even heard about this, it's clear what happened. If we just delete it, people may be confused (if they had somehow heard of this during the registration stage).

Reply:
Yep ok, that makes sense to me too - particularly as we are trying to build the most transparent and logical documentation trail that we can. If it makes sense to a newbie to the project, we've achieved the goal